hawk/app/views/resources/show.html.haml

```diff
@@ -10,121 +10,102 @@
+                = _("Parameters")
                 .pull-right
                   %span.caret
-          #r-metas.panel-collapse.collapse{role: :tabpanel, aria: {labelledby: "r-metacontrol"}}
-            .panel-body
-              %table.table
+          #r-params.panel-collapse.collapse.in{role: :tabpanel, aria: {labelledby: "r-paramcontrol"}}
+            %table.table
+              %tr
+                %th= _("Name")
+                %th= _("Value")
+              - @resource.params.each do |name, value|
                 %tr
-                  %th= _("Name")
-                  %th= _("Value")
-                - @resource.meta.each do |name, value|
-                  %tr
-                    %td= content_tag :code, name
-                    %td= content_tag :code, value
-
-        - if @resource.ops.any?
-          .panel.panel-default
-            .panel-heading#r-opcontrol{role: :tab}
-              %h4.panel-title
-                %a{role: :button, data: {toggle: :collapse, parent: "#r-info"}, href: "#r-ops", aria: {expanded: false, controls: "r-ops"}}
-                  = _("Operations")
-                  .pull-right
-                    %span.caret
-            #r-ops.panel-collapse.collapse{role: :tabpanel, aria: {labelledby: "r-opcontrol"}}
-              .panel-body
-                %table.table
-                  %tr
-                    %th= _("Name")
-                    %th= _("Timeout")
-                    %th= _("Interval")
-                  - @resource.ops.each do |id, op|
-                    %tr
-                      %td= content_tag :code, op["name"]
-                      %td= content_tag :code, op["timeout"] || "-"
-                      %td= content_tag :code, op["interval"] || "-"
+                  %td= content_tag :code, name
+                  %td= content_tag :code, value
+      .panel.panel-default
+        .panel-heading#r-metacontrol{role: :tab}
+          %h4.panel-title
+            %a{role: :button, data: {toggle: :collapse}, href: "#r-metas", aria: {expanded: true, controls: "r-metas"}}
+              = _("Meta Attributes")
+              .pull-right
+                %span.caret
+        #r-metas.panel-collapse.collapse.in{role: :tabpanel, aria: {labelledby: "r-metacontrol"}}
+          %table.table
+            %tr
+              %th= _("Name")
+              %th= _("Value")
+            - @resource.meta.each do |name, value|
+              %tr
+                %td= content_tag :code, name
+                %td= content_tag :code, value
+
+      - if @resource.ops.any?
+        .panel.panel-default
+          .panel-heading#r-opcontrol{role: :tab}
+            %h4.panel-title
+              %a{role: :button, data: {toggle: :collapse}, href: "#r-ops", aria: {expanded: false, controls: "r-ops"}}
+                = _("Operations")
+                .pull-right
+                  %span.caret
+          #r-ops.panel-collapse.collapse{role: :tabpanel, aria: {labelledby: "r-opcontrol"}}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Classes should be listed before IDs (.panel-collapse should precede #r-ops)<br>Line is too long. [92/80]